wolfssl-sys/build.rs

```diff
@@ -40,6 +40,37 @@ fn copy_wolfssl(dest: &str) -> std::io::Result<()> {
     Ok(())
 }
 
+const PATCHES: &[&str] = &[
+    "patches/0001-DoHelloVerifyRequest-only-do-DTLS-1.3-version-check.patch",
+    "patches/0002-DTLS-1.3-move-state-machine-forward-when-HVR-receive.patch",
+    "patches/0003-DtlsShouldDrop-don-t-ignore-app-data-sent-before-a-S.patch",
+    "patches/0004-Dtls13GetRnMask-Correctly-get-chacha-counter-on-BE-s.patch",
+    "patches/1000-Guard-away-properly-don-t-build-sphincs-code.patch",
+    "patches/1001-Allow-easily-disabling-of-SPHINCS.patch",
+    "patches/2000-Rename-utils.c-to-utils.h.patch",
+    "patches/2001-Merge-pull-request-6700-from-julek-wolfssl-dtls13-do.patch",
+];
+
+/**
+ * Apply patch to wolfssl-src
+ */
+fn apply_patch(dest: &str, patch: &str) {
+    let wolfssl_path = format!("{dest}/wolfssl-src");
+
+    let patch_buffer = Command::new("cat")
+        .arg(patch)
+        .stdout(Stdio::piped())
+        .spawn()
+        .unwrap();
+    Command::new("patch")
+        .arg("-d")
+        .arg(wolfssl_path)
+        .arg("-p1")
+        .stdin(Stdio::from(patch_buffer.stdout.unwrap()))
```

Comment:
I think you can drop the `.stdin()` and the needless use of `cat` and instead name the file on the command line i.e. just `.arg(patch)`.

If for some reason the patch really has to be given on stdin please use [`std::fs::File::open`](https://doc.rust-lang.org/std/fs/struct.File.html#method.open) not `cat`.

Reply:
I see, you are not a :cat:  person
Got rid of it now